CHANGELOG.md

```diff
@@ -71,8 +71,7 @@
 
 ### Fixes
 
-- Fixed styling of the incentivized testnet rewards wallet dropdown ([PR 2816](https://github.com/input-output-hk/daedalus/pull/2816/files))
-- Fixed blockchain verification progress text ([PR 2840](https://github.com/input-output-hk/daedalus/pull/2840))
+- Fixed blockain verification progress text ([PR 2840](https://github.com/input-output-hk/daedalus/pull/2840))
```

Comment:
I am not sure about amending Changelog history. I know this was done automatically by git however I am not sure this is the right way..